specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,187 @@
+  ///  - Special keys for browser functions, such as Back, Forward, and Search
+  ///
+  /// This property does not disable accelerator keys related to movement and text editing,
+  /// such as:
+  ///  - Home, End, Page Up, and Page Down
+  ///  - Ctrl-X, Ctrl-C, Ctrl-V
+  ///  - Ctrl-A for Select All
+  ///  - Ctrl-Z for Undo
+  ///
+  /// The exsiting `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` API is a convenience setting 
+  /// for developers to disable all the browser accelerator keys together. This setting sets the default 
+  /// value for the `AllowBrowserAcceleratorHandling` property. 
+  /// By default, `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` is `TRUE` and  
+  /// `AllowBrowserAcceleratorHandling` is `TRUE`. 
+  /// When developers change `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` setting to `FALSE`,  
+  /// this will change default value for `AllowBrowserAcceleratorHandling` to `FALSE`. 
+  /// If developers want specific keys to be handled by the browser after changing the 
+  /// `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` setting to `FALSE`, they need to enable  
+  /// these keys by setting `AllowBrowserAcceleratorHandling` to `TRUE`.
+  /// This API will give the event arg higher priority over the  
+  /// `CoreWebView2Settings.AreBrowserAcceleratorKeysEnabled` setting when we handle the keys.
+  ///
+  /// For browser accelerator keys, when an accelerator key is pressed, the propagation and 
+  /// processing order is: 
+  /// - A CoreWebView2Controller.AcceleratorKeyPressed event is raised
+  /// - WebView2 browser feature accelerator key handling
+  /// - Web Content Handling: If the key combination isn't reserved for browser actions, 
+  /// the key event propagates to the web content, where JavaScript event listeners can 
+  /// capture and respond to it.
+  /// 
+  /// `ICoreWebView2AcceleratorKeyPressedEventArgs` has a `Handled` property, that developers
+  /// can use to marked a key as handled. When the key is marked as handled anywhere along
+  /// the path, the event propagtion stops, and web content will not receive the key.
+  /// With `AllowBrowserAcceleratorHandling` property, when developers mark 
+  /// `AllowBrowserAcceleratorHandling` as `FALSE`, the browser will skip the WebView2 
+  /// browser feature accelerator key handling process, but the event propagation    
+  /// continues, and web content will receive the key combination. 
+
+  /// \snippet ScenarioAcceleratorKeyPressed.cpp
+  /// Gets the `AllowBrowserAcceleratorHandling` property.
+  [propget] HRESULT AllowBrowserAcceleratorHandling([out, retval] BOOL* value);
+
+  /// Sets the `AllowBrowserAcceleratorHandling` property.
+  [propput] HRESULT AllowBrowserAcceleratorHandling([in] BOOL value);
+}
+```
+
+## C#
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+    runtimeclass CoreWebView2AcceleratorKeyPressedEventArgs
+    {
+        Boolean AllowBrowserAcceleratorHandling { get; set; };

```

Comment:
I find the name a little confusing. The name is very general ("Allow Browser Accelerator Handling"), but its meaning is very specific ("Allow the browser to handle this specific key press event"). 

Maybe something more specific:
>AllowBrowserToHandleThisEvent
>AllowBrowserToHandleThisKeyPress

Reply:
I think "AllowBrowserToHandle..." is not quite accurate, because even if you set it to false, the key does go to the web page. It just gets delivered as a plain key and does not have special accelerator powers.

How about `TreatAsAccelerator`? Or if that's confusingly verb-like, `IsTreatedAsAccelerator`.

Reply:
We want to say 'browser' accelerator to match the existing property name in order to distinguish between kinds of accelerators. Some accelerator keys will still be handled as accelerator keys, just not the ones that would be handled by browser specific features like the Browser's DevTools or print dialog accelerator keys.

How about 

* IsTreatedAsBrowserAccelerator

Reply:
I updated the naming to `IsBrowserAcceleratorKeyEnabled`, but if `IsTreatedAsBrowserAccelerator` is better/less confusing for this property, we can change it to `IsTreatedAsBrowserAccelerator`

Reply:
@kmahone @oldnewthing We changed the name of the API to `IsBrowserAcceleratorKeyEnabled`, does that naming better describe this API?